src/features/moreTab/more/MoreScreen.tsx

```diff
@@ -52,6 +56,14 @@ const MoreScreen = () => {
   const navigation = useNavigation<MoreNavigationProp & RootNavigationProp>()
   const spacing = useSpacing()
 
+  useEffect(
+    () =>
+      navigation.addListener('focus', () => {
+        dispatch(fetchAccountSettings())
```

Comment:
This also gets called on app launch right? Is it needed here in case the settings change after it launches? Maybe on another app or something?

Reply:
yeah, it's to keep it updated if settings change from a different device.